integration-tests/chopsticks/src/tests/xcm.test.ts

```diff
@@ -0,0 +1,72 @@
+import { test, beforeAll, afterAll } from 'vitest'
+import { connectParachains, connectVertical, xcmLogger } from '@acala-network/chopsticks'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import * as SpiritnetNetwork from '../network/spiritnet.js'
+import * as PolkadotNetwork from '../network/polkadot.js'
+import * as HydraDxNetwork from '../network/hydroDx.js'
+import type { Config } from '../network/types.js'
+import { keysBob, keysCharlie } from '../helper.js'
+
+let spiritnetContext: Config
+let hydradxContext: Config
+let polkadotContext: Config
+
+beforeAll(async () => {
+	xcmLogger.level = 'info'
+	spiritnetContext = await SpiritnetNetwork.getContext()
+	hydradxContext = await HydraDxNetwork.getContext()
+	polkadotContext = await PolkadotNetwork.getContext()
+
+	await polkadotContext.dev.setStorage(PolkadotNetwork.defaultStorage(keysCharlie.address))
+	await spiritnetContext.dev.setStorage(SpiritnetNetwork.defaultStorage(keysBob.address))
+	await hydradxContext.dev.setStorage(HydraDxNetwork.defaultStorage(keysBob.address))
```

Comment:
Did you mean `keysAlice` in any of these? It's not used anywhere.

Reply:
This was intendant. I wanted to have one account without any balance.